translations/binaries/slps_strings_skills_hidden.json

```
@@ -166,14 +166,14 @@
             "string": "[turquoise]＜ブラスト１消費・連携中[L1_copy][turquoise]押し続け＞[/color][LINE]\n主観的優雅を極める尻待機。[END]",
             "original_bytes_length": 82,
             "available_bytes_size": 87,
-            "translation": "[turquoise]＜블라스트 1 소비・연계 중[L1_copy][turquoise]계속 누르기＞[/color][LINE]\n주관적 우아함을 극한으로 하는 엉덩이대기.[END]"
+            "translation": "[turquoise]＜블라스트 1 소비・연계 중[L1_copy][turquoise]계속 누르기＞[/color][LINE]\n주관적 우아함을 극한으로 하는 엉덩이기.[END]"
```

'尻待機'를 뭐라고 번역하면 좋을까요? 직역하면 '엉덩기 대기'인데 🤔 

기술이니까 엉덩이 치기?라고 의역할 수도 있으려나요

---

공중에서 밑으로 내리찍는 기술이라서 엉덩이 찍기도 괜찮지 않을까요?

---

오, 그렇게 동작하는 기술이었군요. 엉덩이 찍기 좋아보입니다. 👍 

```suggestion
            "translation": "[turquoise]＜블라스트 1 소비・연계 중[L1_copy][turquoise]계속 누르기＞[/color][LINE]\n주관적 우아함을 극한으로 하는 엉덩이 찍기.[END]"
```